app/src/androidTest/java/com/github/se/orator/ui/battle/WaitingForCompletionScreenTest.kt

```diff
@@ -148,32 +149,32 @@ class WaitingForCompletionScreenTest {
     composeTestRule.onNodeWithTag("waitingText").assertIsDisplayed()
     composeTestRule.onNodeWithTag("loadingIndicator").assertIsDisplayed()
 
-    // No navigation should have happened yet since otherUser not completed
+    // No navigation should have happened yet since friend has not completed
     verify(mockNavigationActions, never()).navigateToEvaluationScreen(anyString())
   }
 
   /**
-   * Test navigation when the other user completes the battle. We simulate a battle update where the
+   * Test navigation when the friend completes the battle. We simulate a battle update where the
    * opponentCompleted is now true.
    */
   @Test
   fun testNavigationWhenOtherUserCompleted() {
```

Comment:
Maybe split this into different sub tests so that it's easier to see when one button stopped working

Reply:
I believe there might be a misunderstanding here. This test is focused on checking the navigation and doesn't cover component display verification. Could you clarify if that's what you meant?